libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -145,106 +155,155 @@ export const NavPaperWrapper = styled('div')`
+      grid-template-columns: auto;
+      justify-items: start;
+      width: calc(100% / 6);
+      gap: ${theme.spacing(3)};
+      padding-top: ${theme.spacing(10)};
+      padding-left: ${theme.spacing(4.5)};
     }
   `}
 `
 
 const NavPaper = ({
   main,
   categories,
-  closeMenu
+  closeMenu,
+  children
 }: {
   main: FullNavigationFragment | null | undefined
-  categories: FullNavigationFragment[]
+  categories: FullNavigationFragment[][]
   closeMenu: () => void
+  children: React.ReactNode
 }) => {
   const {
-    elements: {Link, H5}
+    elements: {Link, H4, H6}
   } = useWebsiteBuilder()
   const theme = useTheme()
 
   return (
     <NavPaperWrapper>
+      {children && <NavPaperChildrenWrapper>{children}</NavPaperChildrenWrapper>}
       {!!main?.links.length && (
         <NavPaperMainLinks>
           {main.links.map((link, index) => {
             const url = navigationLinkToUrl(link)
 
             return (
               <Link href={url} key={index} color="inherit" underline="none" onClick={closeMenu}>
-                {link.label}
+                <H4>{link.label}</H4>
               </Link>
             )
           })}
         </NavPaperMainLinks>
       )}
-
       {!!categories.length && (
-        <NavPaperCategoryList>
-          {categories.map(nav => (
-            <NavPaperCategory key={nav.id}>
-              <H5 component="div">{nav.name}</H5>
-
-              <NavPaperCategoryLinks>
-                {nav.links?.map((link, index) => {
-                  const url = navigationLinkToUrl(link)
-
-                  return (
-                    <Link
-                      href={url}
-                      key={index}
-                      color="inherit"
-                      underline="none"
-                      css={navPaperLinkStyling(theme)}
-                      onClick={closeMenu}>
-                      {link.label}
-                    </Link>
-                  )
-                })}
-              </NavPaperCategoryLinks>
-            </NavPaperCategory>
+        <>
+          {categories.map((categoryArray, arrayIndex) => (
+            <LinksGroup key={`category-group-${arrayIndex}`}>
+              {/* Render Separator for every array except the first one */}
+              {arrayIndex > 0 && <Separator />}
+              {categoryArray.map(nav => (
+                <NavPaperCategory key={nav.id}>
+                  <Name>{nav.name}</Name>
+
+                  <NavPaperCategoryLinks>
+                    {nav.links?.map((link, index) => {
+                      const url = navigationLinkToUrl(link)
+
+                      return (
+                        <Link
+                          href={url}
+                          key={index}
+                          color="inherit"
+                          underline="none"
+                          css={navPaperLinkStyling(theme)}
+                          onClick={closeMenu}>
+                          <H6>{link.label}</H6>
```

Comment:
a11y problem as this renders `<h6>`, give it a more appropriate element to render